packages/trader/src/App/Components/Elements/PositionsDrawer/positions-modal-card.tsx

```diff
@@ -102,151 +93,15 @@ const PositionsModalCard = observer(
-                                    'positions-modal-card__indicative--movement-complete': !!result,
-                                })}
-                            >
-                                {status === 'profit' && <Icon icon='IcProfit' />}
-                                {status === 'loss' && <Icon icon='IcLoss' />}
-                            </div>
-                        </div>
-                    </div>
-                    <div className={classNames('positions-modal-card__grid-price-payout')}>
-                        <div className='positions-modal-card__purchase-price'>
-                            <Text size='xxxs' className='positions-modal-card__purchase-label'>
-                                {localize('Purchase price:')}
-                            </Text>
-                            <Text weight='bold' size='xxs' className='positions-modal-card__purchase-value'>
-                                <Money amount={contract_info.buy_price} currency={currency} />
-                            </Text>
-                        </div>
-                        <div className='positions-modal-card__payout-price'>
-                            <Text size='xxxs' className='positions-modal-card__payout-label'>
-                                {localize('Potential payout:')}
-                            </Text>
-                            <Text weight='bold' size='xxs' className='positions-modal-card__payout-value'>
-                                {contract_info.payout ? (
-                                    <Money amount={contract_info.payout} currency={currency} />
-                                ) : (
-                                    <strong>-i</strong>
-                                )}
-                            </Text>
-                        </div>
-                    </div>
-
-                    {result || !!contract_info.is_sold ? (
-                        <PositionsResultMobile
-                            is_visible={!!contract_info.is_sold}
-                            result={result || fallback_result}
-                        />
-                    ) : (
-                        <ProgressSliderMobile
-                            className='positions-modal-card__progress'
-                            current_tick={current_tick}
-                            getCardLabels={getCardLabels}
-                            is_loading={is_loading}
-                            start_time={contract_info.date_start}
-                            expiry_time={contract_info.date_expiry}
-                            server_time={server_time as moment.Moment}
-                            ticks_count={contract_info.tick_count}
-                        />
-                    )}
-                </div>
-            </React.Fragment>
-        );
-
         const contract_vanilla_el = (
             <React.Fragment>
                 <NavLink
-                    className={classNames('dc-contract-card', {
```

Comment:
As according to new design contract card gradient (in mobile) is shown for all contract types (except multipliers), I removed it from Vanillas and add to the rest contracts on line 295-296.